enforcer-parent/enforcer/src/main/java/org/wso2/choreo/connect/enforcer/security/AuthFilter.java

```diff
@@ -181,7 +186,8 @@ private AuthenticationResponse authenticate(Authenticator authenticator, Request
      */
     private void updateClusterHeaderAndCheckEnv(RequestContext requestContext, AuthenticationContext authContext)
             throws APISecurityException {
-
+        log.info("Inside updateCluster env " + requestContext.getProdClusterHeader() + " " +
```

Comment:
This Line gets printed for each and every request right? It is bad. let's make it a debug.

Reply:
fixed